src/main/java/org/ow2/petals/deployer/utils/RuntimeModelComparator.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2018-2019 Linagora
+ *
+ * This program/library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 2.1 of the License, or (at your
+ * option) any later version.
+ *
+ * This program/library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program/library; If not, see http://www.gnu.org/licenses/
+ * for the GNU Lesser General Public License version 2.1.
+ */
+
+package org.ow2.petals.deployer.utils;
+
+import java.util.Collection;
+
+import org.ow2.petals.deployer.runtimemodel.RuntimeComponent;
+import org.ow2.petals.deployer.runtimemodel.RuntimeContainer;
+import org.ow2.petals.deployer.runtimemodel.RuntimeModel;
+import org.ow2.petals.deployer.runtimemodel.RuntimeServiceUnit;
+
+/**
+ * @author alagane
+ */
+public class RuntimeModelComparator {
+    /**
+     * Compare two runtimes models. The URL are not checked.
```

Comment:
A single sentence on the first line.
And write `Compares`. See https://www.oracle.com/technetwork/articles/java/index-137868.html